---
version: "3"
services:
  lavalink:
    container_name: "lavalink"
    image: "ghcr.io/cyb3rgh05t/lavalink:latest"
    restart: "${RESTARTAPP}"
    networks:
      - ${DOCKERNETWORK}
    security_opt:
      - "${SECURITYOPS}:${SECURITYOPSSET}"
    environment:
      - "PGID=${ID}"
      - "PUID=${ID}"
      - "TZ=${TZ}"
    labels:
      - "traefik.enable=true"
      - "traefik.docker.network=proxy"
      - "dockupdater.enable=false"
      - "traefik.http.routers.lavalink-rtr.entrypoints=https"
      - "traefik.http.routers.lavalink-rtr.rule=Host(`lavalink.${DOMAIN}`)"
      - "traefik.http.routers.lavalink-rtr.tls=true"
      - "traefik.http.routers.lavalink-rtr.tls.certresolver=dns-cloudflare"
      - "traefik.http.routers.lavalink-rtr.middlewares=chain-authelia@file"
      - "traefik.http.routers.lavalink-rtr.service=lavalink-svc"
      - "traefik.http.services.lavalink-svc.loadbalancer.server.port=2333"
networks:
  proxy:
    driver: bridge
    external: true
